### PR TITLE
Show which 'game' servers are running

### PR DESCRIPTION
**Pull request by Elijahrane**
Closes https://github.com/space-wizards/SS14.Launcher/issues/67

Requires https://github.com/space-wizards/RobustToolbox/pull/3430 and https://github.com/space-wizards/space-station-14/pull/12347

Draft because I want that up before I go back and polish everything off and make it not look like shit

### CODE REVIEW

**Comment by juliangiebel** [comment]
Shouldn't the default forkId be 'wizden' or 'ss14' instead of 'nyano'? 👀

**Comment by Elijahrane** [comment]
> Shouldn't the default forkId be 'wizden' or 'ss14' instead of 'nyano'? eyes

that's for testing, it will default to empty

**Comment by code-explorer786** [comment]
> Shouldn't the default forkId be 'wizden' or 'ss14' instead of 'nyano'? 👀 

Maybe even `invalid` or even an empty string would be suitable, I guess?

**Review by PJB3005 — CHANGES_REQUESTED** [review]
Shouldn't use forkID for this, it's intended to be an indentifier-style thing, not a human-readable string.

**Comment by Elijahrane** [comment]
uhhh alright, i'll go change the other prs too

File: SS14.Launcher/Views/MainWindowTabs/ServerEntryView.xaml [hunk 1]
```diff
@@ -21,6 +21,7 @@
                      Margin="10, 0" />
           <!-- TODO: Enable text trimming here when Avalonia 0.10 is out of preview and we switch to it. -->
           <TextBlock VerticalAlignment="Center" Text="{Binding Name}" />
+          <TextBlock VerticalAlignment="Center" TextAlignment="Right" Text="{Binding ForkID}" />
```
Comment (PJB3005) [inline SS14.Launcher/Views/MainWindowTabs/ServerEntryView.xaml:24]: This is way too much clutter to fit in the header line.

Comment (Elijahrane) [reply SS14.Launcher/Views/MainWindowTabs/ServerEntryView.xaml:24]: yes, the UI is the most unfinished part of this PR, I am waiting on the other 2 PRs

**Comment by Elijahrane** [comment]
image
this might be ok until we get tags, at some point someone is gonna have to make a bunch of filters and stuff

**Pull request closed without merging**